### ISSUE TITLE
deap for python 3.10

### ISSUE THREAD

**Issue by Edanflame** [DEAP/deap#610]
I am going to upgrade my python from 3.7 to 3.10 in windows and when I try to install deap 1.3.1, it reports error and does not work.  Is the deap for python 3.10 on the schedule? thx.

**Comment by fmder** [comment]
You are right, it seems that the only cp10 wheel has been built for manylinux. I’ll look into it.

**Comment by fmder** [comment]
Ho, I think I tries a couple of months ago, lib_2to3 has been removed from 3.10 ...

**Comment by Edanflame** [comment]
Thank you for your efforts. It helps me a lot.
 I try to setup with deep-master, and error with 'use_2to3 is invalid', so I delete the use_2to3 in setup.py and it works. I make a pr https://github.com/DEAP/deap/pull/620 , and I don't know if it is right.

**Comment by tvdboom** [comment]
any update when a new version will be released that support 3.10?

**Comment by Edanflame** [comment]
https://github.com/DEAP/deap/pull/620 only solves the error during setup, but cannot solve the error during running. So I close it.

**Comment by andreikitaitsev** [comment]
For me the installation worked fine, but a similar error with 'use_2to3 is invalid' and syntax errors in raise statements occured when running the scripts.

**Comment by Edanflame** [comment]
To install deap 1.3.1 in python 3.10, you can degrade the setuptools to 57.0.0 and install with source code

**Comment by ArtemGr** [comment]
For example:

    python -mpip install --user setuptools==58
    python -mpip install --user --upgrade deap

cf. https://stackoverflow.com/questions/69100275/error-while-downloading-the-requirements-using-pip-install-setup-command-use-2

**Comment by meghkc** [comment]
for python 3.10.4
downgrade setuptools to 58.0.0 and use code pip install deap
it will install deap 1.3.1 perfectly
but
while importing base (it says invalid syntax error),  importing tools,creator works fine
are you facing the same please share.

**Comment by aabmets** [comment]
@Edanflame @fmder @tvdboom @andreikitaitsev @mikhailsmilovic @meghkc 

I've created a Python 3.10 port of the DEAP library called [DEAP-er](https://github.com/aabmets/deap-er).
It's currently in beta, but you could check it out and provide feedback about any bugs.
